src/main/java/life/qbic/expdesign/io/MHCLigandDesignReader.java

```diff
@@ -97,58 +97,73 @@ public String[] getMHCClass(String antibody) {
+    sourceMetadata.put(LigandomicsExperimentProperties.Organism.label, OpenbisPropertyCodes.Q_NCBI_ORGANISM.name());
+    sourceMetadata.put(LigandomicsExperimentProperties.Patient_ID.label, OpenbisPropertyCodes.Q_EXTERNALDB_ID.name());
+    sourceMetadata.put(LigandomicsExperimentProperties.Source_Comment.label, OpenbisPropertyCodes.Q_ADDITIONAL_INFO.name());
+    sourceMetadata.put(LigandomicsExperimentProperties.Other_Data_Reference.label, OpenbisPropertyCodes.Q_ADDITIONAL_DATA_INFO.name());
+
+    Map<String, String> extractMetadata = new HashMap<>();
+    extractMetadata.put(LigandomicsExperimentProperties.Tissue.label, OpenbisPropertyCodes.Q_PRIMARY_TISSUE.name());
+    extractMetadata.put(LigandomicsExperimentProperties.Tissue_Comment.label, OpenbisPropertyCodes.Q_ADDITIONAL_INFO.name());
+    extractMetadata.put(LigandomicsExperimentProperties.Detailed_Tissue.label, OpenbisPropertyCodes.Q_TISSUE_DETAILED.name());
+    extractMetadata.put(LigandomicsExperimentProperties.Dignity.label, OpenbisPropertyCodes.Q_DIGNITY.name());
+    extractMetadata.put(LigandomicsExperimentProperties.Cell_Type.label, OpenbisPropertyCodes.Q_TISSUE_DETAILED.name());
+    extractMetadata.put(LigandomicsExperimentProperties.Tumor_Type.label, OpenbisPropertyCodes.Q_TUMOR_TYPE.name());
+    extractMetadata.put(LigandomicsExperimentProperties.Location.label, OpenbisPropertyCodes.Q_TISSUE_LOCATION.name());
+    extractMetadata.put(LigandomicsExperimentProperties.TNM.label, OpenbisPropertyCodes.Q_TUMOR_STAGE.name());
+    extractMetadata.put(LigandomicsExperimentProperties.Metastasis.label, OpenbisPropertyCodes.Q_IS_METASTASIS.name());
+
+    Map<String, String> ligandsMetadata = new HashMap<>();
+    ligandsMetadata.put(LigandomicsExperimentProperties.Antibody.label, OpenbisPropertyCodes.Q_ANTIBODY.name());
+    ligandsMetadata.put(LigandomicsExperimentProperties.MHC_Class.label, OpenbisPropertyCodes.Q_MHC_CLASS.name());
 
     headersToTypeCodePerSampletype = new HashMap<>();
     headersToTypeCodePerSampletype.put(SampleType.Q_BIOLOGICAL_ENTITY, sourceMetadata);
     headersToTypeCodePerSampletype.put(SampleType.Q_BIOLOGICAL_SAMPLE, extractMetadata);
     headersToTypeCodePerSampletype.put(SampleType.Q_TEST_SAMPLE, new HashMap<>());
     headersToTypeCodePerSampletype.put(SampleType.Q_MHC_LIGAND_EXTRACT, ligandsMetadata);
-    // headersToTypeCodePerSampletype.put("Q_MS_RUN", msRunMetadata);
   }
   private void fillParsedCategoriesToValuesForRow(Map<String, Integer> headerMapping,
       String[] row) {
     // logger.info("Collecting possible CV entries for row.");
-    addValueForCategory(headerMapping, row, "Organism");
-    addValueForCategory(headerMapping, row, "Tissue");
-    addValueForCategory(headerMapping, row, "Dignity");
-    addValueForCategory(headerMapping, row, "Tumor Type");
-    addValueForCategory(headerMapping, row, "TNM");
-    addValueForCategory(headerMapping, row, "Antibody");
-    addValueForCategory(headerMapping, row, "MHC Class");
```

Comment:
did you remove some of these values on purpose?

Reply:
Yes, I noticed many of them are actually not controlled by vocabularies and Antibody has a check in the parser itself, so it only makes sense to provide a check for Organism, Tissue and Tumor Type to the wizard.